src/coreclr/vm/gcinfodecoder.cpp

```diff
@@ -1976,7 +1978,7 @@ void GcInfoDecoder::ReportRegisterToGC(
 
         gcFlags |= GC_CALL_PINNED;
     }
-#endif // TARGET_UNIX && !SOS_TARGET_ARM64
+#endif // TARGET_UNIX && !SOS_TARGET_LOONGARCH64
```

Comment:
Also here, should we keep both conditions?

Reply:
Yes, they were incorrect and caused build problems in the diagnostic repo. This section of code is under ifdef TARGET_LOONGARCH64.